kodilla-exception/src/main/java/com/kodilla/exception/test/FlightAnalyst.java

```diff
@@ -0,0 +1,32 @@
+package com.kodilla.exception.test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+
+public class FlightAnalyst {
+    private Map<String,Boolean>flightMap = new HashMap<>();
+
+    public FlightAnalyst() {
+        flightMap.put("Denver",true);
+        flightMap.put("Oslo",false);
+        flightMap.put("Berlin",true);
+        flightMap.put("Paris",false);
+        flightMap.put("Rome",true);
+
+    }
+
+    public void findFlight(Flight flight) throws RouteNotFoundException{
+
+        String arrivalAirport = flight.getArrivalAirport();
+
+        if (flightMap.containsKey(arrivalAirport)){
+             flightMap.entrySet().stream()
+                    .filter(t->t.getKey().equals(arrivalAirport))
```

Comment:
iterujesz po set'cie zamiast wykorzystac mozliwosci hashmapy. Przez to masz zlozonosc obliczeniowa O(n), zamiast O(1). Do wyciagniecia wartosci z mapy uzyj 
flightMap.getValue(arrivalAirport)